packages/framework/src/Component/Grid/GridView.php

```diff
@@ -118,9 +112,7 @@ public function renderBlock($name, array $parameters = [], $echo = true)
     }
 
     /**
-     * @param \Shopsys\FrameworkBundle\Component\Grid\Column $column
      * @param array|null $row
```

Comment:
This should have been removed as well (as the default value is clearly `null`). Seems like an issue in the fixers themselves, it shouldn't influence us as it isn't something critical...

Reply:
It' not the same: `?array` != `array`

Reply:
But argument declared as `array $var = null` is of type `array|null`, not `array`.

Reply:
I see. In that case some NullableFixer/Sniff might handle it

Reply:
You mean `NullableTypeForNullDefaultValueSniff` from Slevomat? That would change the typehint to `?array $var = null`, I think, and that's not intended here.

Reply:
Probably. Personally I prefer `?array $var` without default value, due to clean code